ui/pages/blog_page.py

```diff
@@ -0,0 +1,21 @@
+import allure
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from utils.base_page import BasePage
+
+
+class BlogPage(BasePage):
+    _url = "https://neonevm.org/blog"
+
+    section_header = (By.XPATH, "//h1")
+    header_text = "Blog"
+    author_icon = (By.XPATH, "//div[contains(@class,'120px')][1]//img")
+
+    @allure.step("Check text on the page title")
+    def assert_text_on_blog_page_title(self, text):
+        element = self.wait.until(EC.presence_of_element_located(BlogPage.section_header))
+        assert element.text == BlogPage.header_text
```

Comment:
arg text is unused
you might be missing pre-commit hooks
here's the guide https://neonfoundation.slack.com/archives/C07CVLLU9S4/p1737710673444359

I gues you wanted to do this?
```suggestion
    @allure.step("Check text on the page title")
    def assert_text_on_blog_page_title(self, text):
        element = self.wait.until(EC.presence_of_element_located(BlogPage.section_header))
        assert element.text == text
```

Reply:
Fixed
 ```
@allure.step("Check text on the page title")
    def assert_text_on_blog_page_title(self):
        element = self.wait.until(EC.presence_of_element_located(BlogPage.section_header))
        assert element.text == BlogPage.header_text
```